### DISCUSSION TITLE
Increase timeout for visualizations

### DISCUSSION THREAD

**Discussion by hakube** [apache/superset#18736] [Q&A / Help]
Is there anyway we can increase the timeout? I've already changed the timeout settings in the config.py and in the chartAction.js and I'm still greeted with this error. It's a small dataset btw
![image](https://user-images.githubusercontent.com/38914207/153978710-d8d88a4d-7cce-4129-80ad-031270a068df.png)

**Comment by zhaoyongjie** [comment]
It depends on how you deploy the Superset on your machine. 
1. change `SUPERSEST_WEBSERVER_TIMOUT` in config.py
```
# This is an important setting, and should be lower than your
# [load balancer / proxy / envoy / kong / ...] timeout settings.
# You should also make sure to configure your WSGI server
# (gunicorn, nginx, apache, ...) timeout setting to be <= to this setting
SUPERSET_WEBSERVER_TIMEOUT = int(timedelta(minutes=1).total_seconds())
```

2. change HTTP server timeout settings. For example, Gunicorn uses some flags for worker timeout setting.
```
  -t INT, --timeout INT
                        Workers silent for more than this many seconds are killed and restarted. [30]
  --graceful-timeout INT
                        Timeout for graceful workers restart. [30]
```

**Reply by hakube** [reply]
Hello. Thank you for your response. I've already changed these values but I still get the error when loading my dataset. Not sure why though

**Comment by syh0397** [comment]
What if you solve this problem, Could you give me some tips for me?
